### PR TITLE
Awesome books: plain JavaScript with objectsFeature branch

### PR DESCRIPTION
**Pull request by Mugiira-Kim**
In this exercise I have been able to:

- Create a collection that keeps a list of books.
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection.
- Display all books saved in the collection in the top part of the page.
- Make sure that when a user clicks the "Add" button:
   A new book is added to the collection.
   The new book is displayed in the page.
- Make sure that when a user clicks the "Remove" button:
      The correct book is removed from the collection.
      The correct book dissapears from the page.
- Data is preserved in the browser's memory. 

***I was late to start the module so I am submitting this project solo.**

### CODE REVIEW

**Review by t-yanick — COMMENTED** [review]
Hi @Mugiira-Kim , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the Microvsere [ReadMe template.](https://github.com/microverseinc/readme-template) Make sure you add it and personalize it. It should have all the sections

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by KayLemba — COMMENTED** [review]
Hi @Mugiira-Kim  ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Invalid Code Review Request

You have submitted a project, that does not have the Microvsere [ReadMe template.](https://github.com/microverseinc/readme-template) Make sure you add it and personalize it. It should have all the sections

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Invalid Code Review Request does not count into the code reviews limit.

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @Mugiira-Kim 🙋‍♀️,  

While you made a great effort in this project, I can't review your code.

## Invalid Code Review Request
 
You have submitted a project that has a README that is not professional. Here are key points to consider when documenting your README.

- The template logo should be removed. Optionally, the Microverse logo can be replaced with your own project logo if you have one. ✖️ 
- The only optional section is the FAQ section. If there's no information in the FAQ section then it should be deleted. ✔️ 
- All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present. Even if there's no information for a section or sub-section, rather leave texts like "NA", "None" or "coming soon" than delete it. ✖️ 
***You are missing the `Future Features` section.***
- Future Features section should include 1-3 potential features to be added. ✖️ 
-  Each section description should be removed and replaced with their custom content. The section descriptions are meant to guide you in documenting your README. They are not meant to be part of the final documentation. ✔️ 
- The License suggestion should be removed. ✖️ 
image

- There should not be any blank sections that do not have content. ✖️ 

Kindly document the `Getting Started` section properly. Specify the command and URL for cloning the repo, the command for running the application, and the command for installing dependencies. You can leave the "run tests" and "Deployment" sections with a "None" text for now.
The Section should look like this:

GettingStartedREADME

- There should not be any broken links in the README file. ✖️
***Please fix the MIT license link and make sure that the correct URL is used.***
- Delete the live demo section if the application is not deployed. If project requirements say to deploy the project, the section should be restored and the link updated. ✖️ 

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: .github/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	<title>Awesome Books 2023</title>
+</head>
+<body>
+	<h1>My Book Collection</h1>
+	<form id="add-book-form">
+		<label for="title-input">Title:</label>
+		<input type="text" id="title-input" required>
+		<label for="author-input">Author:</label>
+		<input type="text" id="author-input" required>
+		<button type="submit">Add</button>
```
Comment (joskalenda) [inline .github/index.html:14]: - [ ] Hey @Mugiira-Kim, I reaaly appreciate your code syntax, is is easy to understand, keep the great work dear ❤️ , but I remarked some missing requirements as shown in the image above, it us asked to have a `remove` button in order to remove the item from the collection, of corse I was able to remove the book too but you have added the `eventListener` on the book itself which is not what required, please fix this issue, I know you are smater to do it in a few 😆

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @Mugiira-Kim , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Great syntax
- [ ].. ✅ Green linters
- [ ].. ✅ Good commit messages

## Required Changes ♻️
⚠️ 
- [ ] Checking the image requirement below, this mileston should not be solo, I think you need to work on in in your group pair programming, but you did not, please make sure you partner has contributed on the project too.




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by t-yanick** [comment]
- [ ] Your branch has conflicts and you should resolve that

Screen Shot 2023-02-17 at 4 16 37 AM

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Mugiira-Kim , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- [ ] Your branch is having conflicts that need to be rsolved.
Screen Shot 2023-02-17 at 4 16 37 AM


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @Mugiira-Kim, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**